src/EditorFeatures/Core/Implementation/FindReferences/DefinitionItem.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Completion;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.FindReferences
+{
+    /// <summary>
+    /// Information about a symbol's definition that can be displayed in an editor
```

Comment:
This is supposed to be independent of symbols, but the comments are entirely described in terms of symbols.